src/main/java/com/solvd/movie/service/impl/PgMovieServiceImpl.java

```diff
@@ -19,13 +19,13 @@
 @RequiredArgsConstructor
 public class PgMovieServiceImpl implements PgMovieService {
```

Comment:
@hizmailovich do you have an integration test for covering this piece of code?

Reply:
No :(